src/payment/strategies/clearpay/clearpay-payment-strategy.ts

```diff
@@ -0,0 +1,104 @@
+        private _clearpayScriptLoader: ClearpayScriptLoader
+    ) {}
+
+    async initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+        const { paymentMethods: { getPaymentMethodOrThrow } } = this._store.getState();
+        const paymentMethod = getPaymentMethodOrThrow(options.methodId, options.gatewayId);
+
+        this._clearpaySdk = await this._clearpayScriptLoader.load(paymentMethod);
+
+        return Promise.resolve(this._store.getState());
+    }
+
+    deinitialize(): Promise<InternalCheckoutSelectors> {
+        this._clearpaySdk = undefined;
+
+        return Promise.resolve(this._store.getState());
+    }
+
+    async execute(payload: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const paymentId = payload.payment && payload.payment.gatewayId;
+
+        if (!paymentId) {
+            throw new PaymentArgumentInvalidError(['payment.gatewayId']);
+        }
+
+        const { isStoreCreditApplied: useStoreCredit } = this._store.getState().checkout.getCheckoutOrThrow();
+        let state = this._store.getState();
+        if (useStoreCredit !== undefined) {
+            state = await this._store.dispatch(this._storeCreditActionCreator.applyStoreCredit(useStoreCredit));
+        }
+
+         // Validate nothing has changed before redirecting to Clearpay checkout page
+        await this._checkoutValidator.validate(state.checkout.getCheckout(), options);
+
+        state = await this._store.dispatch(
+            this._paymentMethodActionCreator.loadPaymentMethod(paymentId, options)
+        );
+
+        this._redirectToClearpay(state.paymentMethods.getPaymentMethod(paymentId));
+
+        return Promise.resolve(this._store.getState());
+    }
+
+    async finalize(options: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const state = this._store.getState();
+        const payment = state.payment.getPaymentId();
+        const config = state.config.getContextConfig();
+
+        if (!payment) {
+            throw new MissingDataError(MissingDataErrorType.MissingCheckout);
+        }
+
+        if (!config || !config.payment.token) {
+            throw new MissingDataError(MissingDataErrorType.MissingCheckoutConfig);
+        }
+
+        const paymentPayload = {
+            methodId: payment.providerId,
+            paymentData: { nonce: config.payment.token },
+        };
+
+        await this._store.dispatch(this._orderActionCreator.submitOrder({}, options));
+
+        return this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload));
+    }
+
+    private _redirectToClearpay(paymentMethod?: PaymentMethod): void {
+        if (!this._clearpaySdk || !paymentMethod || !paymentMethod.clientToken) {
+            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+        }
+
+        this._clearpaySdk.initialize({countryCode: this._getCountryCode()});
```

Comment:
why not just...
```suggestion
        this._clearpaySdk.initialize({ countryCode: 'GB' });
```
IDK if clearpay will work in another countries, so you could keep the `_mapCountryToISO2` from afterpay instead